### PR TITLE
New TAB for failsafe configuration

### PR DESCRIPTION
**Pull request by ProDrone**
Rebased-squashed on current development branch.
The modifications from https://github.com/cleanflight/cleanflight/pull/1568 are required for this to work.

Made in cooperation with https://github.com/skaman82/cleanflight-configurator/issues/45

See commit message for a summary of modifications.

### CODE REVIEW

File: main.html [hunk 1]
```diff
@@ -161,36 +163,26 @@
...
             <div id="tabs">
                 <ul class="mode-disconnected">
-                    <li class="tab_landing"><a href="#" i18n="tabLanding" class="tabicon ic_welcome"
-                        title="Welcome"></a></li>
-                    <li class="tab_help"><a href="#" i18n="tabHelp" class="tabicon ic_help"
-                        title="Documentation & Support"></a></li>
-                    <li class="tab_firmware_flasher"><a href="#" i18n="tabFirmwareFlasher"
-                        class="tabicon ic_flasher" title="Firmware Flasher"></a></li>
+                    <li class="tab_landing"><a href="#" i18n="tabLanding" class="tabicon ic_welcome" title="Welcome"></a></li>
+                    <li class="tab_help"><a href="#" i18n="tabHelp" class="tabicon ic_help" title="Documentation & Support"></a></li>
```
Comment (hydra) [inline main.html]: &amp;

Comment (hydra) [inline main.html]: never mind, was just a formatting in your PR.  I will fix via another commit after merging.

**Comment by hydra** [comment]
Good work @skaman82 and @ProDrone 

I gave this a little test on my machine here, no problems with adjusting any of the values.

**Comment by ProDrone** [comment]
@hydra You made a remark that there is styling in `failsafe.html` and it should be in `failsafe.css`.

I just checked the other `*.html` files and found that most of them have many `style` keywords.
Is this a requirement or dependency for  the pop-up helptexts @skaman82 ? Or is just that it needs to be fixed in all of them?

**Comment by skaman82** [comment]
@ProDrone The remark was in main.html regarding the html code for the "&" character. It should be "&amp;". I don't se any other comments and I can not follow your question. What do you mean by style keywords? Do you mean the combination of different CSS classes?

**Comment by ProDrone** [comment]
@skaman82 Sorry for not being clear. I think the remark was made on IRC. I checked all `*.html` files to search for element styling using the keyword `style = ...`, i am currently moving all styling out of the `*.html` and into the `*.css` files...

**Comment by skaman82** [comment]
ok, now I can follow! :) Yes, I made some styling inline, but just for unique elements which are used just once in that case. I saw that before in the HTML files, so I just repeated :). Thanks for doing the clean up. I could done this my self if necessary - its my mess after all :)

**Comment by ProDrone** [comment]
@skaman82 
Well, it is indeed giving me a headache :)
I modified failsafe and some other already. Would be nice if you want to pick it up from there...
Pushed to: https://github.com/ProDrone/cleanflight-configurator/tree/move_styling_to_css_prodrone

**Comment by ProDrone** [comment]
@skaman82 Did you pickup the code from this branch so that i can delete it?

**Comment by skaman82** [comment]
Not yet.

**Comment by ProDrone** [comment]
I like to provide you another branch without the failsafe tab files (i separated and continued to work on them). Is that okay with you?

**Comment by ProDrone** [comment]
@skaman82 I removed the previous one, use this branch instead please:
https://github.com/ProDrone/cleanflight-configurator/tree/move_styling_to_css_v2

**Pull request merged**